samples/mnist/serving/tensorflow/serving_client.py

```diff
@@ -0,0 +1,61 @@
+# TensorFlow and tf.keras
+import tensorflow as tf
```

Comment:
Looks good except the fact that now user has to install these libraries in his/her machine.

Reply:
Agree. Per request last time, if we want to show an image, seems right now this is a straightforward solution. I will check if we can simplify it using container as you suggest.